core/src/main/resources/com/graphhopper/custom_models/bike.json

```diff
@@ -1,6 +1,6 @@
 // to use this custom model you need to set the following option in the config.yml
 // graph.elevation.provider: srtm   # enables elevation
-// graph.encoded_values: bike_priority, bike_access, roundabout, bike_average_speed, bike_road_access, foot_road_access, average_slope, mtb_rating, hike_rating, country, road_class
+// graph.encoded_values: bike_priority, bike_access, bike_network, roundabout, bike_average_speed, bike_road_access, foot_road_access, average_slope, mtb_rating, hike_rating, country, road_class
```

Comment:
I just tested this branch with my existing config and am a little bit confused now: adding of the  `bike_network` to `graph.encoded_values` in config.yml is actually not needed. You also didn't add it to config-example.yml. Can you please explain why this is not needed? As it is not needed, would it be possible to just don't add `bike_network` here?